nltk/test/gluesemantics_malt_fixt.py

```diff
@@ -4,6 +4,6 @@ def setup_module():
     from nltk.parse.malt import MaltParser
 
     try:
-        depparser = MaltParser("maltparser-1.7.2")
+        depparser = MaltParser("maltparser")
```

Comment:
I would have preferred to have a fix that involves no longer requiring `MaltParser` to take a `parser_dirname` parameter. It can become optional, but right now the flow is as follows:
* MaltParser gets initialized with a mandatory `parser_dirname`.
* This `parser_dirname` is used to find the `MaltParser` executable. If this fails, then the `MALT_PARSER` environment variable is looked for. If this also fails, then an error is thrown. 

However, this means that if a user wishes to initialize `MaltParser` with an environment variable (which is how I would prefer it), then the user must also still give a path to `MaltParser`? That is weird. Perhaps we can make `parser_dirname` optional, allowing for 2 ways to initialize the parser.